docs/en/setup/backend/backend-fetcher.md

```diff
@@ -122,3 +122,21 @@ kafka-fetcher:
       enable.auto.commit: true
       ...
 ```
+
+When use Kafka MirrorMaker 2.0 to replicate topics between Kafka clusters, you can set the source Kafka Cluster alias(mm2SourceAlias) and separator(mm2SourceSeparator) according to your Kafka MirrorMaker config.
```

Comment:
typo.
mirror marker.
I recommend to add link refer to kafka mirror docs.

Reply:
Done.